shared/actions/fs/index.js

```diff
@@ -34,17 +34,20 @@ function* listFavoritesSaga(): Saga.SagaGenerator<any, any> {
 }
 
 const direntToMetadata = (d: RPCTypes.Dirent) => ({
-  name: d.name,
+  name: d.name.split('/').pop(),
```

Comment:
are we sure there aren't backslashes here on Windows?

Reply:
FWIW I tested it on my Windows VM and it worked.